controller/verificationController.js

```diff
@@ -0,0 +1,76 @@
+const User = require("../models/usersModel");
+const { sendVerificationEmail } = require("../email");
+
+// повторне висилання мейла на верифікацію
+const verificationEmailResend = async (req, res, next) => {
+  const { email } = req.body;
+
+  if (!email) {
+    return res.status(400).json({
+      status: "error",
+      code: 400,
+      message: "Missing required field email",
+    });
+  }
+
+  try {
+    const user = await User.findOne({ email });
+    if (!user) {
+      return res.status(404).json({
+        status: "error",
+        code: 404,
+        message: "User not found",
+      });
+    }
+    if (user.verify) {
+      return res.status(400).json({
+        status: "error",
+        code: 400,
+        message: "Verification has already been passed",
+      });
+    }
+
+    const verificationToken = user.verificationToken;
+    await sendVerificationEmail(email, verificationToken);
```

Comment:
W tej funkcji zrobiłem sugestię żeby użyć throw. Wówczas jeśli będzie błąd to tutaj zostanie przesłany i w tym bloku catch wysłany dalej.